example/separable_compilation/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+KOKKOS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+KOKKOS_INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
+
+KOKKOS_ADD_EXECUTABLE(
+  example_separable_compilation
+  SOURCES cmake_example.cpp functor.cpp
+)
+
+add_test(NAME KokkosSeparableCompilation COMMAND Kokkos_example_separable_compilation 10)
```

What's that trailing `10` argument?
```suggestion
add_test(NAME KokkosSeparableCompilation COMMAND Kokkos_example_separable_compilation)
```

---

Nevermind